projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/publicize-connection-test-results.php

```diff
@@ -122,7 +122,11 @@ public function get_items( $request ) { // phpcs:ignore VariableAnalysis.CodeAna
 			}
 		}
 
-		if ( 'linkedin' === $item['id'] && 'must_reauth' === $test_result['connectionTestPassed'] ) {
+		if (
+			isset( $item['id'] )
```

Comment:
While this will fix the warning, the logic here still seems weird. Why is it looking at only the last element of `$items`? Should this check be inside the loop just above instead of after it?

Reply:
Looks like I'm not the first to ask that: https://github.com/Automattic/jetpack/pull/11280/files#r257089706